src/Symfony/Bundle/FrameworkBundle/Tests/Console/Descriptor/AbstractDescriptorTest.php

```diff
@@ -241,11 +241,22 @@ private function getEventDispatcherDescriptionTestData(array $objects)
         $data = array();
         foreach ($objects as $name => $object) {
             foreach ($variations as $suffix => $options) {
-                $description = file_get_contents(sprintf('%s/../../Fixtures/Descriptor/%s_%s.%s', __DIR__, $name, $suffix, $this->getFormat()));
+                $description = file_get_contents($this->getFixtureFilename($name.'_'.$suffix));
                 $data[] = array($object, $description, $options);
             }
         }
 
         return $data;
     }
+
+    private function getFixtureFilename($name)
+    {
+        $format = $this->getFormat();
+        $file = ($baseDir = __DIR__.'/../../Fixtures/Descriptor').'/'.$name.'.'.$format;
```

Comment:
can you move the `$baseDir` assignement to its own statement ? Nested assignments are not that readable.

Reply:
Done.